### PR TITLE
Multiple bug fixes to run parsec benchmark

### PR DESCRIPTION
**Pull request by lqr8**

### CODE REVIEW

**Comment by shinezyy** [comment]
Clean solution for TLB priv level !

**Review by shinezyy — APPROVED** [review]

File: src/cpu/o3/fetch.cc [hunk 1]
```diff
@@ -835,7 +835,7 @@ Fetch::finishTranslation(const Fault &fault, const RequestPtr &mem_req)
         // If we have, just wait around for commit to squash something and put
         // us on the right track
         if (!cpu->system->isMemAddr(mem_req->getPaddr())) {
-            warn("Address %#x is outside of physical memory, stopping fetch, %lu\n",
+            DPRINTF(Fetch, "Address %#x is outside of physical memory, stopping fetch, %lu\n",
```
Comment (tastynoob) [inline src/cpu/o3/fetch.cc:838]: Use "warn" can help us find the fetch stuck easier

Comment (shinezyy) [reply src/cpu/o3/fetch.cc:838]: > Use "warn" can help us find the fetch stuck easier

But if on the speculative path, might it be too verbose?

@notlqr Did you observe a too verbose print here?

Comment (lqr8) [reply src/cpu/o3/fetch.cc:838]: Some benchmarks produce too much warns here

**Pull request merged**